spec/shoryuken/environment_loader_spec.rb

```diff
@@ -4,6 +4,36 @@
 RSpec.describe Shoryuken::EnvironmentLoader do
   subject { described_class.new({}) }
 
+  describe '#load' do
+    before do
+      Shoryuken.groups.clear
+      # See issue: https://stackoverflow.com/a/63699568 for stubbing AWS errors
+      allow(Shoryuken::Client)
+        .to receive(:queues)
+        .with('stubbed_queue')
+        .and_raise(Aws::SQS::Errors::NonExistentQueue.new(nil, nil))
+      allow(subject).to receive(:load_rails)
+      allow(subject).to receive(:prefix_active_job_queue_names)
+      allow(subject).to receive(:require_workers)
+      allow(subject).to receive(:validate_workers)
+      allow(subject).to receive(:patch_deprecated_workers)
+      Shoryuken.options[:groups] = [['custom', { queues: ['stubbed_queue'] }]]
+    end
+
+    context "when given queues don't exist" do
+      specify do
+        expect { subject.load }.to raise_error(
+          ArgumentError,
+          <<-MSG.gsub(/^\s+/, '')
```

Comment:
Ha, the `~` operator is actually used to deal with indentation. The suggested / cleanest solution I could find was to use `#gsub`. I know it can be expensive though (since it's re-creating string objects), but this doesn't seem like a potential blocker since this command is invoked on load versus runtime.

Reply:
This solution looks good! Like you said, this isn't a hot path, so the performance won't matter much.